R/PMP.R

```diff
@@ -0,0 +1,315 @@
+#'Estimate the Probable Maximum Precipitation based on Herschfield Methodology
+#'
+#'PMP Estimation based on Hershfield methodology from Daily values
+#'@param val.max        Vector with annual maximum.
+#'@param input.hr      PMP to be defined 24 hrs, 6 hrs or 1 hr.
+#'@param area.km     Watershed area. This methodology is for Western US. By default, if the correction is not needed is recommended a value of 1.

```

Comment:
Can you be clearer how this is for western US? How does that apply to Canada? The function name itself may have to include the regional specifications.

Reply:
That the point that it maybe confusing, and I am not clear what is the best way to present. PMP has an area reduction (as a point); however, when there are important areas more than 20 km2 (if I remember well), it can be expected an area reduction; by default, that was implemented in the western US. So, It can be applicable to Canada but some specific validation need to be required. it maybe will be useful just to remove the factor. In any case, it should be more conservative without it.

Reply:
I think you should remove the factor OR provide greater documentations in the function.